docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,59 @@
+
+## The difference between core and example modules
+
+**Core modules** provide generic, reusable functionality that benefits a wide range of customers. They offer a foundation for customization and serve as versatile building blocks.
+
+**Example Modules** in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
+
+## Example module use-cases
+
+### SCOS: Showcase functionality
+
+SCOS uses example modules to fill the gap with concrete/example implementation over an abstract workflow (latter is the target of delivery), where the concrete/example implementation is never actually usable by any concrete customer or business use-case, thus the actual business logic is not part of the core (eg: the React API Example).
+
+### SCOS: Unique business use-case
+
+SCOS uses example modules to implement functionality that is unique business (customer) use-case specific. The actual business logic is not part of the core as it does NOT offer regular core quality and is extracted into an example module.
+
+## Obligations
+
+### Exclusions and Non-Commitments
+
+Sprkyer has no strict obligations regarding the maintenance, compatibility, flexibility, and extendibility of the example modules.
```

Comment:
I have no good definition on the title, but the text should be more like the following

```
We believe in providing transparency and setting clear expectations for our open-source packages.
We want you to know that our approach is a bit different on example modules:

* Stability
** We aim to keep our packages in good shape, but we don't promise a stable release.

* Compatibility
** While we work to keep things compatible, we can't guarantee forward or backward compatibility.

* Learnability
** We provide documentation when we can, but it's not a firm commitment. We encourage community learning and support.

...

In a nutshell, think of our example packages as a playground - there's potential for fun and innovations -, explore at your own pace!

```